build.sbt

```diff
@@ -555,6 +555,16 @@ lazy val finagleBackend = (projectMatrix in file("finagle-backend"))
   .jvmPlatform(scalaVersions = scala2)
   .dependsOn(core % compileAndTest)
 
+lazy val armeriaBackend = (projectMatrix in file("armeria-backend"))
+  .settings(commonJvmSettings)
+  .settings(testServerSettings)
+  .settings(
+    name := "armeria-backend",
+    libraryDependencies += "com.linecorp.armeria" % "armeria" % "1.4.0"
+  )
+  .jvmPlatform(scalaVersions = List(scala2_13))
```

Comment:
I suppose we can support all scala 2 & scala 3 versions?

Reply:
I used `FutureConverters`, and since it's located in different packages in versions prior to 2.13 i didn't add them here